docs/content/docs/concepts/summaryFormats.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Summary and snapshot formats
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run build:md-magic" from the docs folder. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE:path=../../../../packages/runtime/container-runtime/src/summary/summaryFormats.md&start=2) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+## Table of contents
```

Comment:
@Rick-Kirkham Do you think this doc is too technical to be on ff.com? I have been debating whether to put this entire doc or a high level version of it that only highlights a simple summary. It could point to this document on Github for details.